src/index.ejs

```diff
@@ -60,7 +60,7 @@
           useModal: useModal,
           buttonId: 'onfido-button',
           onReady: function() {
```

Comment:
it gives us some freedom in case we don't want to render the sdk straight away in the future, but since we already have people depending on the fact that init is synchronous (see #153), it might be worth to drop it and assume we will always do a synchronous init...